docs/UserGuide.md

```diff
@@ -18,7 +24,7 @@ especially for those who are teaching multiple classes/modules at the same time.
 
 ## Quick start
 
-1. Ensure you have [Java `11`](https://www.oracle.com/java/technologies/downloads/#java11) or above installed in your Computer.
+1. Before getting started with TAPA, ensure you have [Java `11`](https://www.oracle.com/java/technologies/downloads/#java11) or above installed on your computer. You can check the version of Java on your computer by inputting `java -version` in your [terminal](https://www.ionos.com/help/email/troubleshooting-mail-basicmail-business/access-the-command-prompt-or-terminal/#:~:text=Open%20Command%20Prompt%20in%20Windows,cmd%22%20and%20then%20click%20OK.).
```

Comment:
@Benjykoh Do you think this link will help a reader access terminal?

Reply:
@brelkh Yup I think this would be great!